src/Service/Aws.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Service;
+
+use Aws\S3\S3Client;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+
+class Aws
+{
+    private $awsKey;
+    private $secret;
+    private $bucket;
+    private $prefix;
+
+    public function __construct(string $awsKey = null, string $secret = null, string $bucket = null, string $prefix = null)
+    {
+        $this->awsKey = $awsKey;
+        $this->secret = $secret;
+        $this->bucket = $bucket;
+        $this->awsPrefix = $prefix;
+    }
+
+    public function filesystem () {
```

Comment:
This file should probably be split  into two, one for managing the files and the second part to configure the adapter (and injected into the first).